src/magic_folder/test/test_config.py

```diff
@@ -0,0 +1,166 @@
+from shutil import (
+    rmtree,
+)
+from os import (
+    mkdir,
+)
+
+from twisted.python.filepath import (
+    FilePath,
+)
+
+from testtools import (
+    ExpectedException,
+)
+from testtools.matchers import (
+    Equals,
+    NotEquals,
+)
+
+from .common import (
+    SyncTestCase,
+)
+from ..config import (
+    create_global_configuration,
+    load_global_configuration,
+)
+from ..snapshot import (
+    create_local_author,
+)
+
+
+class TestGlobalConfig(SyncTestCase):
+
+    def setUp(self):
+        super(TestGlobalConfig, self).setUp()
+        self.temp = FilePath(self.mktemp())
+
+    def tearDown(self):
+        super(TestGlobalConfig, self).tearDown()
+        rmtree(self.temp.path)
+
+    def test_create(self):
+        create_global_configuration(self.temp, "tcp:1234", "tcp:localhost:5678")
```

Comment:
It's supposed to return a Deferred that succeeds with 0, maybe that's an okay assertion to make here?  Although it's a bit on the weak side.  Maybe this doesn't add much on top of the other happy-path tests - `test_load_db`, `test_rotate_api_key`, etc?

Reply:
This now tests different filenames (and `create_global_configuration` is synchronous).